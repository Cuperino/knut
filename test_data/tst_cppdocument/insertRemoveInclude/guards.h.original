// Some comments

#ifndef GUARD_H
#define GUARD_H

class Foo {

};

#endif
