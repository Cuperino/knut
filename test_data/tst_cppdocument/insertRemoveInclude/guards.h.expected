// Some comments

#ifndef GUARD_H
#define GUARD_H

#include <bar.h>

class Foo {

};

#endif
